### ISSUE TITLE
Can't parse mutations

### ISSUE THREAD

**Issue by Simon-Ince** [ivelum/graphql-py#14]
When parsing a mutation I get:

``` 
File "./internal/gqlparser/parser.py", line 166, in p_operation_definition8
    p[0] = self.operation_cls(p[1])(selections=p[2])
TypeError: __init__() missing 1 required positional argument: 'name'
```

I can see the `GraphQLParser` class has `operation_cls` which is looking for `operation_type == 'mutation':` so I assume this module is built to parse mutations as well as queries?

I can't see a mutation example in the tests or README, is here something I'm missing? Or should this work?

**Comment by Simon-Ince** [comment]
I've figured out what I was doing wrong.

Shorthand is supported for queries but not mutations, once I added a mutation name it worked.

Maybe this is a feature request and not a bug? Or maybe the query example in the README could not be a shorthand example to help with the confusion?

**Comment by stebunovd** [comment]
hey @Simon-Ince, thank you for the bug report. I think I've misinterpreted the GraphQL spec. The previous version assumed that only queries can be anonymous, but I've read it carefully once again and it turns out that it was wrong. The bug is now fixed in v0.8.1 which is already on PyPI
